CHANGELOG.md

```diff
@@ -1,8 +0,0 @@
-# Changelog
```

Comment:
We should either use it or remove it. I'm planning to keep the changelog in our release notes.

Reply:
yeah we don't need the changelog. The releases page can auto generate the diff for us